core/chains/evm/client/sub_forwarder.go

```diff
@@ -36,7 +36,7 @@ func newSubForwarder[T any](destCh chan<- T, interceptResult func(T) T, intercep
 // start spawns the forwarding loop for sub.
 func (c *subForwarder[T]) start(sub ethereum.Subscription, err error) error {
 	if err != nil {
-		close(c.srcCh)
+		close(c.destCh)
```

Comment:
Was this a pre-existing bug?

Reply:
Look like that was changed by accident? Doesn't seem to effect any test cases. Changed back.